zuul.d/molecule.yaml

```diff
@@ -263,6 +263,17 @@
       - ^molecule-requirements.txt
       - ^ci_framework/roles/run_hook/(?!meta|README).*
       - ^ci/playbooks/molecule.*
+- job:
+    name: cifmw-molecule-set_openstack_containers
+    parent: cifmw-molecule-base
+    nodeset: centos-9-crc-xl
```

Comment:
hmmm I don't think you added the job to the `NEED_CRC_XL` variable located in `scripts/create_role_molecule` did you?
It would be overridden if anyone regenerate the jobs using the script.